src/context/AppContext.js

```diff
@@ -72,26 +71,13 @@ const AppProvider = ({children}) => {
       setUser(data.user)
     }
   })
-  
-  // mutations
-    // const EVENTS_MUTATION = gql`
-    //   mutation {
-    //     createEvent(input: {
-    //       tourId: 1,
-    //       name: "TEST",
-    //       venueId: 3,
-    //       startTime: "2021-04-22",
-    //       endTime: "2021-04-22"
-    //     })
-    //     {
-    //       name
-    //     }
-    //  `
-  // const [ helperFunuction ] = useMutation(EVENTS_MUTATION)
+
+  //MUTATIONS
+  const [ mutateEvent, {error} ] = useMutation(EVENTS_MUTATION)
```

Comment:
We were so close yesterday! Thanks for muscling through this and figuring it out.

Reply:
Whoa! useMutation? What is this sorcery? 😂